saleor/graphql/order/tests/mutations/test_order_note_add.py

```diff
@@ -88,10 +88,22 @@ def test_order_add_note_fail_on_empty_message(
     order_id = graphene.Node.to_global_id("Order", order_with_lines.id)
     variables = {"id": order_id, "message": message}
     response = staff_api_client.post_graphql(
-        ORDER_ADD_NOTE_MUTATION, variables, permissions=[permission_manage_orders]
+        ORDER_NOTE_ADD_MUTATION, variables, permissions=[permission_manage_orders]
     )
     content = get_graphql_content(response)
-    data = content["data"]["orderAddNote"]
+    data = content["data"]["orderNoteAdd"]
     assert data["errors"][0]["field"] == "message"
     assert data["errors"][0]["code"] == OrderErrorCode.REQUIRED.name
     order_updated_webhook_mock.assert_not_called()
+
+
+def test_order_note_add_fail_on_missing_permission(staff_api_client, order):
+    order_id = graphene.Node.to_global_id("Order", order.id)
+    variables = {"id": order_id, "message": "a note"}
+    response = staff_api_client.post_graphql(ORDER_NOTE_ADD_MUTATION, variables)
+    content = get_graphql_content(response, ignore_errors=True)
+    assert len(content["errors"]) == 1
+    assert (
+        content["errors"][0]["message"]
+        == "You need one of the following permissions: MANAGE_ORDERS"
+    )
```

Comment:
There is `assert_no_permission` in `saleor/graphql/tests/utils.py` for such checks. Also missing `given, when, then`.